packages/api/src/logger/index.ts

```diff
@@ -0,0 +1,200 @@
+import pino, {
+  BaseLogger,
+  DestinationStream,
+  LevelWithSilent,
+  LoggerOptions,
+  redactOptions,
+} from 'pino'
+
+/**
+ * Determines if log environment is development or test
+ *
+ * Boolean
+ */
+const isDevelopment =
```

Comment:
I'm wondering if it would make sense to split out test and dev? I can imagine that in test you might not want the noise of the logger in between.

Reply:
Good point.

Test's default could in fact be "silent" which is [an option](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/561d92092281eb4ad3cebea7d7e4a4647a7881b1/types/pino/index.d.ts#L611):

```ts
    type Level = 'fatal' | 'error' | 'warn' | 'info' | 'debug' | 'trace';
    type LevelWithSilent = Level | 'silent';
```

Or maybe `error`?

Which default do you think?

Reply:
I _think_ `error` would make the most sense. Most use cases I can think of when I'd have `logger.error` is when I really want to know when something weird happened. If I'm debugging stuff then I'd normally use `console.log`. 

Let's see if others have strong opinions on this, if not let's go with `error`

Reply:
@peterp and @Tobbe agree to have a special default case for test and that be error? What would you expect test to log when mocking and testing services?

Note one would have to declare a new logger if using it directly in test code.